bin/cli.js

```diff
@@ -17,15 +18,19 @@ argv
     flags[flagKey] = keyValue[1] || true;
   });
 
+// @todo refactor
 let url;
 if (flags.expectations) {
-  const expectationsConfig = getConfig(flags.expectations);
+  const expectationsConfig = getExpectationsConfig(flags.expectations);
   url = expectationsConfig.url;
   flags.metrics = expectationsConfig.metrics;
-} else {
-  url = argv.filter(f => !f.startsWith('-')).shift();
+} else if (flags.submit) {
+  flags.sheet = getSheetsConfig(flags.submit);
+  delete flags.submit;
```

Comment:
Another think. I'll just leave it here. It's about flags `submit`, `expectations`. Right now for both of them users have to pass file path as value. What if we can do it better, like
`pwmetrics --submit --config=./path-to-config`
`pwmetrics --expecations --config=./path-to-config`
if path is not passed then config will be read from `package.json`.
Format of config file might be
```js
{
  url: 'http://example.com/',
  expectations: {
    metrics: {
      ttfcp: {
        warn: '>=30',
        error: '>=5000'
      },
      ttfmp: {
        warn: '>=3000',
        error: '>=50'
      },
      psi: {
        warn: '>=1500',
        error: '>=32'
      }
    }
  },
  sheets: {
    type: 'GOOGLE_SHEETS',
    options: {
      spreadsheetId: '1Uo4a4KQTaC1FBDPMIJ6xLMmOd2yLyps-6jBC5x4x6I8',
      tableName: 'data'
    }
  }
}

```